### PR TITLE
Correct error when the call is down when stopping an output

### PR DESCRIPTION
**Pull request by benlangfeld**

### CODE REVIEW

File: lib/punchblock/translator/asterisk/component/stop_by_redirect.rb [hunk 1]
```diff
@@ -12,8 +12,12 @@ def execute_command(command)
             if @complete
               command.response = ProtocolError.new.setup 'component-already-stopped', "Component #{id} is already stopped", call_id, id
             else
-              stop_by_redirect Punchblock::Event::Complete::Stop.new
-              command.response = true
+              begin
+                stop_by_redirect Punchblock::Event::Complete::Stop.new
+                command.response = true
+              rescue ChannelGoneError
+                command.response = ProtocolError.new.setup :item_not_found, "Could not find a call with ID #{call_id}", call_id
```
Comment (benlangfeld) [inline lib/punchblock/translator/asterisk/component/stop_by_redirect.rb]: This needs a test

**Comment by sfgeorge** [comment]
Fantastic, thank you @benlangfeld!  I'm going to try to give this a quick spin.

**Comment by sfgeorge** [comment]
This is working fabulously in my integration testing;  The error is reported at TRACE level, successfully handled, and even the related call channel is preserved and able to continue operation in Punchblock.  Fantastic work!!!

**Comment by benlangfeld** [comment]
I'll merge this as soon as I get CI working again, and also propose it against Adhearsion 3.

**Pull request merged**